### PR TITLE
Enhancements to test source view

### LINKED ISSUE #1576: Enhancements to test source view
1. Rename: "Update time" -> "Git commit time"
2. Add "refresh" button, that will query for test versions once again
3. May be add information (hash code) of the latest commit

### PR DESCRIPTION
**Pull request by Cheshiriks**
**What's done:**
- Renamed: "Update time" -> "Git commit time"
- Added "refresh" button, that will query for test versions once again

Closes #1576

### CODE REVIEW

**Comment by sanyavertolet** [comment]
@Cheshiriks I cannot quite understand what has changed by the screenshot, could you please provide `What's done` block in PR?

**Review by sanyavertolet — APPROVED** [review]
lgtm

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt [hunk 1]
```diff
@@ -123,6 +123,22 @@ private fun organizationTestsMenu() = FC<OrganizationTestsMenuProps> { props ->
             setTestSuitesSourceSnapshotKeys(testSuitesSourceSnapshotKeys.filterNot(it::equals))
         }
     }
+    val refreshTestSuitesSourcesSnapshotKey = useDeferredRequest {
+        val response = get(
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt:127]: it looks like as a code duplication with line 52-61. Can you please extract common part to a common method and re-use it

**Comment by akuleshov7** [comment]
Not lgtm, please make a normal button with icon

**Review by nulls — CHANGES_REQUESTED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt]: Fixed

**Review by nulls — APPROVED** [review]

**Pull request merged**